<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoDetailsPage</class>
 <widget class="QWidget" name="VideoDetailsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Details - VibeTube</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <layout class="QVBoxLayout" name="leftPanel" stretch="0,0,1">
     <item>
      <widget class="QGroupBox" name="groupVideoInfo">
       <property name="styleSheet">
        <string notr="true">QGroupBox { background-color: #ffe6f0; border-radius: 8px; margin-top: 8px;color: black;  }</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="videoInfoLayout">
        <item>
         <widget class="QLabel" name="lblTitle">
          <property name="styleSheet">
           <string notr="true">font-size: 22px; font-weight: bold; color: #5e3c58;</string>
          </property>
          <property name="text">
           <string>Video Title </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblViews">
          <property name="text">
           <string>Views: 1200 | Genre: Travel</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="uploaderLayout">
          <item>
           <widget class="QLabel" name="lblUploader">
            <property name="text">
             <string>Uploaded by: Channel Name (Joined via FK)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSubscribe">
            <property name="styleSheet">
             <string notr="true">background-color: #f6a5c0; padding: 4px 12px; border-radius: 6px; font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Subscribe</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="actionsLayout">
       <item>
        <widget class="QPushButton" name="btnLike">
         <property name="styleSheet">
          <string notr="true">background-color: #ffb6c1; padding: 8px 15px; border-radius: 6px; font-weight: bold;</string>
         </property>
         <property name="text">
          <string>üëç Like </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupComments">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox { background-color: #ffe6f0; border-radius: 8px; }</string>
       </property>
       <property name="title">
        <string>Comments</string>
       </property>
       <layout class="QVBoxLayout" name="commentsLayout">
        <item>
         <layout class="QHBoxLayout" name="commentInputLayout">
          <item>
           <widget class="QLineEdit" name="txtComment">
            <property name="placeholderText">
             <string>Write a comment...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPostComment">
            <property name="styleSheet">
             <string notr="true">background-color: #f497a8; color: white; padding: 6px; border-radius: 6px; font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Post</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listComments"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightPanel">
     <item>
      <widget class="QGroupBox" name="trending_box">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox { background-color: #fddde6; border-radius: 8px; }</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="suggestedLayout">
        <item>
         <widget class="QLabel" name="trending_lbl">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Trending Videos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_trending">
          <property name="maximumSize">
           <size>
            <width>4456</width>
            <height>1234</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="back_video">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Back to searchpage</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
